### ISSUE TITLE
ipdb in pyramid app

### ISSUE THREAD

**Issue by delijati** [gotcha/ipdb#77]
ipdb is trying to read the development.ini
```
Starting server in PID 30852.
serving on http://0.0.0.0:6543
WARNING: Attempting to work in a virtualenv. If you encounter problems, please install IPython inside the virtualenv.
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/development.ini", line 6
    [app:main]
        ^
SyntaxError: invalid syntax

2016-02-16 14:42:18,425 ERROR [waitress:339][waitress] Exception when serving /form
Traceback (most recent call last):
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/waitress/channel.py", line 336, in service
    task.service()
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/waitress/task.py", line 169, in service
    self.execute()
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/waitress/task.py", line 388, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/router.py", line 223, in __call__
    response = self.invoke_subrequest(request, use_tweens=True)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/router.py", line 198, in invoke_subrequest
    response = handle_request(request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid_debugtoolbar-2.4.2-py2.7.egg/pyramid_debugtoolbar/toolbar.py", line 192, in toolbar_tween
    response = _handler(request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid_debugtoolbar-2.4.2-py2.7.egg/pyramid_debugtoolbar/panels/performance.py", line 57, in resource_timer_handler
    result = handler(request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/tweens.py", line 20, in excview_tween
    response = handler(request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid_tm-0.12.1-py2.7.egg/pyramid_tm/__init__.py", line 101, in tm_tween
    reraise(*exc_info)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid_tm-0.12.1-py2.7.egg/pyramid_tm/__init__.py", line 83, in tm_tween
    response = handler(request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/router.py", line 145, in handle_request
    view_name
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/view.py", line 541, in _call_view
    response = view_callable(context, request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/config/views.py", line 353, in rendered_view
    result = view(context, request)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/pyramid-1.6.1-py2.7.egg/pyramid/config/views.py", line 481, in _class_requestonly_view
    response = inst()
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/PYRA_WORK/views/form.py", line 61, in __call__
    import ipdb
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/ipdb/__init__.py", line 16, in <module>
    from ipdb.__main__ import set_trace, post_mortem, pm, run, runcall, runeval, launch_ipdb_on_exception
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/ipdb/__main__.py", line 66, in <module>
    ipapp.initialize()
  File "<decorator-gen-111>", line 2, in initialize
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/IPython/terminal/ipapp.py", line 312, in initialize
    self.init_code()
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/IPython/core/shellapp.py", line 296, in init_code
    self._run_cmd_line_code()
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/IPython/core/shellapp.py", line 422, in _run_cmd_line_code
    self.exit(1)
  File "/home/delicj/develop/work/MONEY/PYRA_WORK/env/local/lib/python2.7/site-packages/traitlets/config/application.py", line 579, in exit
    sys.exit(exit_status)
SystemExit: 1
```

**Comment by gotcha** [comment]
Is this a bug in the new 0.8.2 ?

**Comment by delijati** [comment]
Yes its the 0.8.2. It works fine with 0.8.1. I tested it also with a simple pyramid app `pcreate -t alchemy foo`.

**Comment by gotcha** [comment]
Can you check you still have the issue with 0.8.3 ?

**Comment by delijati** [comment]
Thanks @gotcha its fixed in 0.8.3
